bin/mail/2017-01-05-go-beyond-an-hour/generate-list

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# To be sent ~ 2pm on Thurs, 2017-01-05
+
+# 3 categories:
+# 1) us_hoc_no_account - US 2016 Hour of Code organizers who do not have Code Studio accounts
+# 2) international_hoc_no_account - international 2016 Hour of Code organizers who do not have Code Studio accounts
+# 3) teachers_no_pd - US Teachers with Code Studio accounts that have not gone through our PD
+
+# In all cases, skip all @schools.nyc.gov, @strongschools.nyc, and @cps.edu emails.
+# These will be sent separate emails later.
+
+@results = {}
+@category_counts = Hash.new(0)
+
+def create_temp_teachers_table
+  PEGASUS_DB.run "
```

Comment:
This hits the master DB? Can we instead make this happen on the replica? Also with the associated `DROP TABLE` call.